src/main/java/Duke.java

```diff
@@ -1,10 +1,88 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static final String DIVIDING_LINE = "________________________________________";
+    public static final int MAX_STORED_TASKS = 100;
+    public static final int TODO_OFFSET = 5;
+    public static final int DEADLINE_OFFSET = 9;
+    public static final int EVENT_OFFSET = 6;
```

Comment:
I like how you simplified reading by declaring these constants in uppercase with underscore to separate words. It helped increase readability later on.